ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -806,6 +807,12 @@ private AdminResponse handleBlobStoreControlRequest(DataInputStream requestStrea
               if (storageManager.controlCompactionForBlobStore(partitionId, true)) {
                 error = ServerErrorCode.No_Error;
                 logger.info("Store is successfully started and functional for partition: {}", partitionId);
+                List<PartitionId> failToUpdateList =
+                    storageManager.setBlobStoreStoppedState(Arrays.asList(partitionId), false);
```

Comment:
minor: Can you use `Collections.singletonList()` here instead of `Arrays.asList()`?

Reply:
done. Thanks for suggestion.